cmd/webrpc-test/main.go

```diff
@@ -30,6 +32,9 @@ var (
 	timeoutFlag = serverFlags.Duration("timeout", time.Minute, "exit after given timeout")
 )
 
+//go:embed api.ridl
+var apiScheme string
```

Comment:
I think we can move this `//go:embed` to the `tests` pkg, so we don't need to move the file